include/csoundCore.h

```diff
@@ -582,6 +582,7 @@ typedef struct CORFIL {
                               (used by opcodes) */
     MYFLT   *spin;         /* offset into csound->spin */
     MYFLT   *spout;        /* offset into csound->spout, or local spout */
+    int      spout_flag;
```

Comment:
Is this necessary? I did a search through the code and found spout_flag was only ever set to 0 (unless I'm missing something).

Reply:
removed.